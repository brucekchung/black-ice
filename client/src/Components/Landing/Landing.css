.Landing {
  margin: 0 auto;
  height: 100%;
  display: flex;
  flex-direction: column;
  background: navy;
  color: white;
}

.Landing a {
  text-decoration: none;
  color: white;
  font-size: 1.4em;
}

.Landing h2 {
  display: block;
  text-align: center;
  margin: 50px;
}

.Landing .selection {
  border: 1px solid red;
  display: flex;
  flex-direction: column;
  padding: 20px;
  width: 21%;
  text-align: center;
}

.Landing .center-links {
  margin: 0 auto;
  display: flex;
  justify-content: space-between;
  width: 70%;
}

.Landing img {
  height: 70px;
  margin-top: 20px;
}
