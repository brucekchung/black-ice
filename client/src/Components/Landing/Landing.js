import React, { Component } from 'react'
import Nav from '../Nav/Nav'

class Landing extends Component {
  constructor() {
    super()
    this.state = {
      user: 'Ulyana'
    }
  }

  render() {
    return (
      <div>
        <Nav />
        <h2>Welcome, { this.state.user }!</h2>
        <a href="/DataEntry">Add Data</a>
        <a href="/Reports">Reports</a>
        <a href="/ViewAll">Edit Data</a>
      </div>
    )
  }
}

export default Landing
