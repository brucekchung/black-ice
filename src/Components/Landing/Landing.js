import React, { Component } from 'react'
import Nav from '../Nav/Nav'

class Landing extends Component {
  constructor() {
    super()
    this.state = {}
  }


  render() {
    return (
      <div>
        <Nav />
        Landing Component
      </div>
    )
  }
}

export default Landing
