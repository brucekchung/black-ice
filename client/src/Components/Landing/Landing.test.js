import React from 'react'
import Landing from './Landing'
import { shallow } from 'enzyme'

describe('Landing', () => {
  let wrapper;

  beforeEach(() => {
    const wrapper = shallow(<Landing />)
  })

  it('should match snapshot upon load', () => {
    expect(wrapper).toMatchSnapshot()
  })

})