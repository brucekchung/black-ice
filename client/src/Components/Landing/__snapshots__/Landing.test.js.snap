// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing should match snapshot upon load 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Landing />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Nav />,
        <h2>
          Welcome, 
          Ulyana
          !
</h2>,
        <a
          href="/DataEntry"
>
          Add Data
</a>,
        <a
          href="/Reports"
>
          Reports
</a>,
        <a
          href="/ViewAll"
>
          Edit Data
</a>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Welcome, ",
            "Ulyana",
            "!",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Welcome, ",
          "Ulyana",
          "!",
        ],
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Add Data",
          "href": "/DataEntry",
        },
        "ref": null,
        "rendered": "Add Data",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Reports",
          "href": "/Reports",
        },
        "ref": null,
        "rendered": "Reports",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Edit Data",
          "href": "/ViewAll",
        },
        "ref": null,
        "rendered": "Edit Data",
        "type": "a",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Nav />,
          <h2>
            Welcome, 
            Ulyana
            !
</h2>,
          <a
            href="/DataEntry"
>
            Add Data
</a>,
          <a
            href="/Reports"
>
            Reports
</a>,
          <a
            href="/ViewAll"
>
            Edit Data
</a>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Welcome, ",
              "Ulyana",
              "!",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Welcome, ",
            "Ulyana",
            "!",
          ],
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Add Data",
            "href": "/DataEntry",
          },
          "ref": null,
          "rendered": "Add Data",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Reports",
            "href": "/Reports",
          },
          "ref": null,
          "rendered": "Reports",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Edit Data",
            "href": "/ViewAll",
          },
          "ref": null,
          "rendered": "Edit Data",
          "type": "a",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
